test/browser/lifecycle.test.js

```diff
@@ -1999,6 +2000,14 @@ describe('Lifecycle methods', () => {
 	});
 
 	describe('#componentDidCatch', () => {
+
+		/** @type {Error} */
+		let expectedError;
```

Comment:
@rpetrich FYI, I've updated the error boundary tests a bit. Lemme know if you have any thoughts!

Reply:
I think the biggest change is specifying that throwing refs can be caught in the component that defines them

Reply:
Updated error boundary tests look good and check more behaviour 👍